packages/@ember/object/observable.ts

```diff
@@ -435,15 +280,137 @@ const Observable = Mixin.create({
+    key: keyof this,
+    target: Target,
+    method: ObserverMethod<Target, this>
+  ): this;
+  removeObserver(key: keyof this, method: ObserverMethod<this, this>): this;
+
+  // NOT TYPE SAFE!
+  /**
+    Set the value of a property to the current value plus some amount.
+
+    ```javascript
+    person.incrementProperty('age');
+    team.incrementProperty('score', 2);
+    ```
+  */
+  incrementProperty(keyName: keyof this, increment?: number): number;
+
+  // NOT TYPE SAFE!
+  /**
+    Set the value of a property to the current value minus some amount.
+
+    ```javascript
+    player.decrementProperty('lives');
+    orc.decrementProperty('health', 5);
+    ```
+  */
+  decrementProperty(keyName: keyof this, decrement?: number): number;
+
+  // NOT TYPE SAFE!
+  /**
+    Set the value of a boolean property to the opposite of its
+    current value.
+
+    ```javascript
+    starship.toggleProperty('warpDriveEngaged');
+    ```
+  */
+  toggleProperty(keyName: keyof this): boolean;
+
+  /**
+   * Returns the cached value of a computed property, if it exists.
+   * This allows you to inspect the value of a computed property
+   * without accidentally invoking it if it is intended to be
+   * generated lazily.
+   */
+  cacheFor<K extends keyof this>(key: K): unknown;
+}
+const Observable = Mixin.create({
+  get(keyName: string) {
+    return get(this, keyName);
+  },
+
+  getProperties(...args: string[]) {
+    return getProperties(this, ...args);
+  },
+
+  set(keyName: string, value: unknown) {
+    return set(this, keyName, value);
+  },
+
+  setProperties(hash: object) {
+    return setProperties(this, hash);
+  },
+
+  /**
+    Begins a grouping of property changes.
+
+    You can use this method to group property changes so that notifications
+    will not be sent until the changes are finished. If you plan to make a
+    large number of changes to an object at one time, you should call this
+    method at the beginning of the changes to begin deferring change
+    notifications. When you are done making changes, call
+    `endPropertyChanges()` to deliver the deferred change notifications and end
+    deferring.
+
+    @method beginPropertyChanges
+    @return {Observable}
+    @private
+  */
+  beginPropertyChanges() {
+    beginPropertyChanges();
+    return this;
+  },
+
+  /**
+    Ends a grouping of property changes.
+
+    You can use this method to group property changes so that notifications
+    will not be sent until the changes are finished. If you plan to make a
+    large number of changes to an object at one time, you should call
+    `beginPropertyChanges()` at the beginning of the changes to defer change
+    notifications. When you are done making changes, call this method to
+    deliver the deferred change notifications and end deferring.
+
+    @method endPropertyChanges
+    @return {Observable}
+    @private
+  */
+  endPropertyChanges() {
+    endPropertyChanges();
+    return this;
+  },
+
+  /**
```

Comment:
This comment needs to be moved to the interface.

Reply:
I've moved the missing comment, thanks!